.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  integ-tests:
+    runs-on: ubuntu-latest
+    environment:
+      name: prod
+    steps:
+        - uses: actions/checkout@v3
+        - uses: actions/setup-python@v5
+          with:
+            python-version: '3.11'
+        - name: allows us to build arm64 images
+          run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+        - name: run integration tests
+          run: sudo make integ-tests-with-docker
```

Comment:
do we (or why do we) need to run it with `sudo` ?

Reply:
It doesn't work without it. I ran into some permissions issue when I tried. Even on my dev desktop I have to use it.
```
make integ-tests
make[1]: Entering directory `/local/home/mbfreder/Tooling/aws-lambda-runtime-interface-emulator'
python3 -m venv .venv
Error: [Errno 13] Permission denied: '/local/home/mbfreder/Tooling/aws-lambda-runtime-interface-emulator/.venv/bin/activate.csh'
```